stock_landed_cost_backdating/models/stock_landed_cost.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Ecosoft Co., Ltd (https://ecosoft.co.th)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class StockLandedCost(models.Model):
+    _inherit = "stock.landed.cost"
+
+    def _update_valuation_backdating(self):
+        for valuation in self.stock_valuation_layer_ids:
+            accounting_date = self.account_move_id.date
+            if accounting_date:
+                self._cr.execute(
```

Comment:
_cr is deprecated since a long time.

Use self.env.cr instead

Reply:
Noted I will change in this PR #1411 :+1: